ci/scripts/execute_redis_tests.sh

```diff
@@ -19,12 +19,14 @@
 
 cd ..
 
-# We are currently using a personal fork for this repo because our code does not implement all
+# We are currently using a patched version of this repo because our code does not implement all
 # Redis commands.  Once all commands needed to run relevant test files are implemented, we hope to
-# use Redis's repo instead.
-git clone --config transfer.fsckObjects=false https://github.com/prettyClouds/redis.git
+# use Redis's repo without a patch.
+git clone --config transfer.fsckObjects=false https://github.com/redis/redis.git
+cp geode-redis/src/acceptanceTest/resources/0001-configure-redis-tests.patch redis
 cd redis
-git checkout tests-geode-redis
+git checkout origin/5.0
+git am < 0001-configure-redis-tests.patch
```

Comment:
It seems like the application of the patch is failing.  Could be related to this error:
```
*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: empty ident name (for <geode@heavy-lifter-bc46dab9-e3b2-587d-b59e-a07a7b894615.c.apachegeode-ci.internal>) not allowed
```
...though I tried removing my identity and running it locally with a fresh clone of redis and it worked.  Maybe an identity is required to apply a patch on heavy-lifter?

Reply:
Ok, after hijacking the pipeline, if you set
```
git config --global user.email "you@example.com"
git config --global user.name "Your Name"
```
before trying to apply the patch, it will be applied successfully.  Not sure what you should set it to though?

Reply:
thanks @sabbey37! I'll do some more testing on that

Reply:
Switched to using `git apply` which obviates the need to configure an email.